v0/destinations/hs/HSTransform-v2.js

```diff
@@ -382,6 +428,9 @@ const batchEvents = destEvents => {
     } else if (operation === "updateContacts") {
       // Identify: making chunks for CRM update contact endpoint
       updateContactEventsChunk.push(event);
+    } else if (event.message.endpoint.includes("associations")) {
```

Comment:
Instead of classifying via an endpoint, we can classify it via some parameter as we are doing for the rest of the operations.

Reply:
updated to use the operation